playwright.config.ts

```diff
@@ -1,4 +1,21 @@
 import { defineConfig, devices } from '@playwright/test';
+import { execSync } from 'child_process';
+
+// Get the current branch
+function getCurrentBranch(): string {
```

Comment:
Q: This seems a bit out of scope for this ticket, personally I feel this should be its own separate ticket.  Unless this is specifically here to work alongside your tests, I would consider removing these changes.

Reply:
While I agree that it is out of scope for this, we talked about it in discord and me making these changes. What say you, @shashilo?

Reply:
If it's needed to run your test, it's in scope.